rts/at/util/src/main/java/org/jboss/jbossts/star/util/logging/at18NLogger.java

```diff
@@ -0,0 +1,124 @@
+package org.jboss.jbossts.star.util.logging;
+
+import static org.jboss.logging.Logger.Level.ERROR;
+import static org.jboss.logging.Logger.Level.WARN;
+
+import javax.transaction.xa.XAException;
+
+//import javax.json.JsonObject;
+
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * at18N log messages for the rest-at module.
+ */
+@MessageLogger(projectCode = "AT")
+public interface at18NLogger {
```

Comment:
please rename the class to `atI18NLogger`

Reply:
+1

Reply:
Done